doc/release/1.5.0-notes.rst

```diff
@@ -39,39 +57,230 @@ New features
+
+There have been several improvements to `scipy.optimize.linprog`:
 
+- The ``linprog`` benchmark suite has been expanded considerably.
+- ``linprog``'s dense pivot-based redundancy removal routine and sparse 
+  presolve are faster
+- When ``scikit-sparse`` is available, solving sparse problems with 
+  ``method='interior-point'`` is faster
+
+The caching of values when optimizing a function returning both value and 
+gradient together has been improved, avoiding repeated function evaluations 
+when using a ``HessianApproximation`` such as ``BFGS``.
+
+``differential_evolution`` can now use the modern ``np.random.Generator`` as 
+well as the legacy ``np.random.RandomState`` as a seed.
 
 `scipy.signal` improvements
 ---------------------------
+A new optional argument ``include_nyquist`` is added to ``freqz`` functions in 
+this module. It is used for including the last frequency (Nyquist frequency).
 
+`scipy.signal.find_peaks_cwt` now accepts a ``window_size`` parameter for the 
+size of the window used to calculate the noise floor.
 
 `scipy.sparse` improvements
 ---------------------------
+Outer indexing is now faster when using a 2d column vector to select column 
+indices.
+
+`scipy.sparse.lil.tocsr` is faster
+
+Fixed/improved comparisons between pydata sparse arrays and sparse matrices
+
+BSR format sparse multiplication performance has been improved.
 
 `scipy.spatial` improvements
 ----------------------------
+`scipy.spatial.geometric_slerp` has been added to enable geometric 
+spherical linear interpolation on an n-sphere
+
+`scipy.spatial.SphericalVoronoi` now supports calculation of region areas in 2D 
+and 3D cases
+
+New ``cKDTree`` benchmarks are available for building and querying of balanced/
+unbalanced kd-trees.
+
+`scipy.special` improvements
+--------------------------
+The following functions now have Cython interfaces in `cython_special`:
+
+- `scipy.special.erfinv`
+- `scipy.special.erfcinv`
+- `scipy.special.spherical_jn`
+- `scipy.special.spherical_yn`
+- `scipy.special.spherical_in`
+- `scipy.special.spherical_kn`
+
+`scipy.special.log_softmax` has been added to calculate the logarithm of softmax 
+function. It provides better accuracy than ``log(scipy.special.softmax(x))`` for 
+inputs that make softmax saturate.
 
 `scipy.stats` improvements
 --------------------------
+The function for generating random samples in `scipy.stats.dlaplace` has been 
+improved. The new function generates random samples by calculating the 
+difference between two independent and identically distributed geometric 
+distributions. This new function is approximately twice as fast as the previous 
+(default) function which generated samples by inverting the CDF. This new 
+function also uses less memory than the previous function. For one sample, the 
+new function uses ~25% less memory; for one million samples, it uses ~40% less 
+memory; for ten million samples, it uses ~60% less memory.
```

Comment:
I'd suggest to shorten this a little. Something like ".... uses between 25% and 60% less memory (see `gh-xxxxx`)"